packer/terraform/builders.tpl

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "${kind}",
+  "type": "amazon-ebs",
+  "access_key": "{{user `aws_access_key`}}",
+  "secret_key": "{{user `aws_secret_key`}}",
+  "region": "eu-west-1",
+  "subnet_filter": {
+    "filters": {
+      "tag:Usage": "packer_build"
+    },
+    "most_free": true,
+    "random": false
+  },
+  "vpc_id": "${vpc_id}",
+  "instance_type": "t2.micro",
+  "source_ami_filter": {
+    "filters": {
+      "virtualization-type": "hvm",
+      "name": "ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-*",
+      "root-device-type": "ebs"
+    },
+    "owners": ["099720109477"],
```

Comment:
Should we hardcode this value?

Reply:
Yes, we should. It's the canonical owner id.

Reply:
magic numbers FTW